test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup_teardown/README.md

```diff
@@ -0,0 +1,37 @@
+aws_ssm_integration_test_setup_teardown
+=========
+
+This role will create infrastructure for aws_ssm connection plugin, after integration test will get completed it will tear down infrastructure.
+
+Requirements
+------------
+
+Ansible machine shoould AWS permission for all the resources used for aws_ssm connection plugin intrgration testing.
```

Comment:
Please verify all spellings in the readme. You can install spellcheck addon into your editor.
I am using this: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
Updated Readme